lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -15,6 +17,18 @@
 Applies DSC Resources by generating a configuration file and applying it.
 EOT
 
+  def initialize(value={})
+    super(value)
+    set_symlink_folder
+  end
+
+  def set_symlink_folder
+    puppetModulesVendorFolder = "#{ENV['ProgramFiles']}\\WindowsPowerShell\\Modules\\PuppetVendoredModules"
```

Comment:
We were preferring Program Files as that is still machine but considered where you install modules. It's the same place that PowerShell-Get puts modules. Once I'm ready for this to be reviewed more I will send a note.